### PR TITLE
feat: 模版添加面包屑组件

### PR DESCRIPTION
**Pull request by xrr2016**
## Why

解决了面包屑导航的问题

## How

image

## Docs

https://www.yuque.com/loyulc/sovgiy/owfgdw#8YIOW

### CODE REVIEW

File: template/modules/breadcrumb/store/bread.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const commitSetBreads = async breadcrumb => {
+      const paths = routePath.split('/')
+      const allPromise = breadcrumb.map(async (item, index) => {
+        const path = paths.slice(0, index + 2).join('/')
+        const matchComps = this.$router.getMatchedComponents(path)
+        let to = ''
+
+        if (item.action) {
+          const name = await dispatch(item.action, route)
+          item.name = name ? name : ''
```
Comment (snowlocked) [inline template/modules/breadcrumb/store/bread.js]:
```suggestion
          item.name = name || ''
```

File: template/modules/breadcrumb/store/bread.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      const breads = await Promise.all(allPromise)
+      commit('setBreads', breads)
+    }
+
+    if (meta && meta.breadcrumb) {
+      commitSetBreads(meta.breadcrumb)
+    } else {
+      const found = breadData.find(item => item.name === name)
+
+      if (found) {
```
Comment (snowlocked) [inline template/modules/breadcrumb/store/bread.js:51]:
```suggestion
      found && commitSetBreads(found.breadcrumb)
```

**Review by snowlocked — APPROVED** [review]

**Review by donaldshen — APPROVED** [review]

File: template/modules/breadcrumb/store/bread.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  breads(state) {
+    return state.breads.filter(item => item.name)
+  },
+}
+
+export const actions = {
+  async generateBreadcrumb({ commit, dispatch }, route) {
+    const { path: routePath, name, meta } = route
+
+    const commitSetBreads = async breadcrumb => {
```
Comment (levy9527) [inline template/modules/breadcrumb/store/bread.js]: setBreads

**Pull request merged**